go.mod

```diff
@@ -5,6 +5,7 @@ go 1.14
 require (
 	github.com/apache/dubbo-go v1.5.5
 	github.com/apache/dubbo-go-hessian2 v1.7.0
+	github.com/dubbogo/dubbo-go-proxy-filter v0.1.0-rc1.0.20210120132524-c63f4eb13725 //TODO
 	github.com/dubbogo/go-zookeeper v1.0.2
 	github.com/emirpasic/gods v1.12.0
```

Comment:
split import

Reply:
If every thing ok I will release the `dubbo-go-proxy-filter` and update this.